pages/index.tsx

```diff
@@ -21,7 +20,7 @@ interface SSProps {
 
 const starmapsGithubUrl = 'https://github.com/pln-planning-tools/Starmap/blob/main/User%20Guide.md';
 
-export async function getServerSideProps(): Promise<{props: SSProps}> {
+export async function getServerSideProps(): Promise<{ props: SSProps }> {
```

Comment:
looks like a lot of issues that should have been failing in the linter, or left out of this PR.